finder-app/writer.c

```diff
@@ -0,0 +1,105 @@
+/* Assignment 2 - AESD       
+ * writer.c
+ * Author: Saloni Shah */
+
+#include<stdio.h>
+#include<string.h>
+#include<syslog.h>
+#include<stdlib.h>
+#include<dirent.h>
+#include<libgen.h>
+#include<sys/time.h>
+#include<sys/types.h>
+#include<sys/stat.h>
+#include<fcntl.h>
+#include<errno.h>
+#include<unistd.h>
+
+int main(int argc, char *argv[] ) {
+
+	int fd, wr, cs, statdir, mkdr;
+	struct stat st = {0};
+	char *dir;
+	char path[200]="\0";
```

Comment:
@saloni1307 how do you know 200 bytes is long enough for path?  You could use `PATH_MAX` in `limits.h` instead or dynamically allocate.